recipes/tqdm/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: tqdm
+  version: "4.7.1"
+
+source:
+  fn: tqdm-4.7.1.tar.gz
+  url: https://files.pythonhosted.org/packages/a6/74/f4bf4f44f68a8dcba74db6892bbcebb3e78edcab0f5d5640d5054a43a67b/tqdm-4.7.1.tar.gz
+  md5: 0d3bda7f215e406d9bfa66c65bb6eaf7
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  entry_points:
+    - tqdm=tqdm._main:main
+
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - tqdm
+
+  commands:
+    - tqdm --help
+
+  requires:
+    - coverage
+    - flake8
+    - nose
+
+
+about:
+  home: https://github.com/tqdm/tqdm
+  license: Mozilla Public License 2.0 (MPL 2.0) or MIT License
```

Comment:
MPL is more similar to GPL. So, dual licensing with MIT provides people the option to fork the codebase and use it for private development.